packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,7 +302,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: dropdownWidth || this._dropDown.clientWidth - 2 } }>
```

Comment:
I'm seeing that if there is enough options to make a scroll bar, the dropdown moves next to the input 

![image](https://user-images.githubusercontent.com/1434956/28976337-501f790e-78f2-11e7-92f0-5235c259b27c.png)